commons/error_messages.py

```diff
@@ -52,8 +52,8 @@
                                "operation: This copy request is illegal because it is trying to "
                                "copy an object to itself without changing the object's metadata, "
                                "storage class, website redirect location or encryption attributes.")
-RGW_HEAD_OBJ_ERR = ("An error occurred (404) when calling the HeadObject operation: Not Found")
-CORTX_HEAD_OBJ_ERR = ("An error occurred (404) when calling the HeadObject operation: Not Found")
+RGW_HEAD_OBJ_ERR = "An error occurred (404) when calling the HeadObject operation: Not Found"
+CORTX_HEAD_OBJ_ERR = "An error occurred (404) when calling the HeadObject operation: Not Found"
```

Comment:
If error message is same in both rgw and cortx, we dont need separate variables

Reply:
@archanabpatil , this is only placeholder.
When HeadObject API is validated on cortx-rgw, we will make changes accordingly.

Reply:
okay.